beacon_chain/state/state_transition.py

```diff
@@ -1,3 +1,26 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    NewType,
+    Tuple,
+    Type,
+    TYPE_CHECKING,
+    Union,
+)
```

Comment:
`Any` and `Type` are not used right now (but `Any` might if `dict` is changed to `Dict[str, Any]`)